polling_stations/apps/data_importers/management/commands/import_eoni_from_s3.py

```diff
@@ -420,15 +434,22 @@ def call_import_eoni_command(self, options):
         }
 
         if options.get("send_slack_report"):
-            if self.dc_environment == "production":
-                channel = BOTS_CHANNEL
-            else:
-                channel = BOTS_TESTING_CHANNEL
-
-            importer_opts["slack"] = channel
+            importer_opts["slack"] = self.slack_channel
 
         eoni_importer.handle(**importer_opts)
 
+    def send_failure_to_slack(self, exception: Exception):
+        slack_client = SlackClient(channel=self.slack_channel)
+        try:
+            response = slack_client.send_message(
+                message=":warning: *import_eoni_from_s3 command failed*",
+            )
+            slack_client.send_message(
+                message=f"Error: {str(exception)}", thread_ts=response.get("ts")
+            )
+        except Exception as slack_e:
+            self.stderr.write(f"Error posting to Slack: {str(slack_e)}")
```

Comment:
I think we want to `raise slack_e` here?

Reply:
Yes, your right here. Then the command will be marked as failed in the AWS command runner console,.